.github/workflows/main.yml

```diff
@@ -110,6 +110,21 @@ jobs:
       - name: check docs
         run: tools/generate_docs.sh
 
+  alpine-linux:
+    name: alpine 3.11.6 (musl)
+    runs-on: ubuntu-18.04
+    container: alpine:3.11.6
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          submodules: recursive
+      - name: install tools
+        run: apk update && apk add build-base cmake git linux-headers
+      - name: configure
+        run: cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_BACKEND=ON -DBUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_PREFIX=install -Bbuild/release -S.
+      - name: build
+        run: cmake --build build/release --target install -- -j2
```

Comment:
You could also add running the two unit tests since it's quick anyway, and you see if things link and work.